src/main/java/io/github/debuggyteam/tablesaw/TableSaw.java

```diff
@@ -23,58 +24,57 @@
-        
-        // Receives serverside notice that the tablesaw craft button is clicked
-        ServerPlayNetworking.registerGlobalReceiver(TABLESAW_CHANNEL, new TableSawServerReceiver());
-        
-        // Updates recipes on the server when data is loaded or reloaded
-        ResourceLoader.get(ResourceType.SERVER_DATA).registerReloader(new TableSawResourceLoader());
-        
-        // Syncs recipes for all connected players when a live reload happens
-        ResourceLoaderEvents.END_DATA_PACK_RELOAD.register((server, resourceManager, error) -> {
-            // Server will be null here if this is the first reload as the game is starting. In that case there are no players to notify.
-            if (server == null) return;
-            for(ServerPlayerEntity player : server.getPlayerManager().getPlayerList()) {
-                TableSawRecipeSync.syncFromServer(server, player);
-            }
-        });
-        
-        // Syncs recipes when a player connects
-        ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
-            TableSawRecipeSync.syncFromServer(server, handler.getPlayer());
-        });
-    }
-    
-    
+	public static final String MODID = "tablesaw";
+	public static final Identifier TABLESAW_CHANNEL = identifier("tablesaw");  // S <-> C
+	public static final Identifier RECIPE_CHANNEL = identifier("recipe_sync"); // S -> C only
+	public static final int MESSAGE_ENGAGE_TABLESAW = 0x120000;
+	
+	public static final Logger LOGGER = LoggerFactory.getLogger("TableSaw");
+	
+	public static final TableSawBlock TABLESAW = new TableSawBlock(QuiltBlockSettings.of(Material.WOOD).nonOpaque());
+	
+	/** Register a sound for the tablesaw to use */
+	public static final Identifier TABLESAW_SFX = new Identifier(MODID, "tablesaw_sfx");
+	//public static final SoundEvent TABLESAW_SOUND_EVENT = new 
+	
+	public static final ScreenHandlerType<TableSawScreenHandler> TABLESAW_SCREEN_HANDLER = new ScreenHandlerType<>((syncId, inventory) -> new TableSawScreenHandler(syncId, inventory, ScreenHandlerContext.EMPTY), FeatureFlagBitSet.empty());
+	
+	/** Creates an identifier with this mod as the namespace */
+	public static Identifier identifier(String path) {
+		return new Identifier(MODID, path);
+	}
+	
+	@Override
+	public void onInitialize(ModContainer mod) {
+		LOGGER.info("Hello Quilt world from {}!", mod.metadata().name());
+		
+		Registry.register(Registries.SCREEN_HANDLER_TYPE, new Identifier(MODID, "tablesaw"), TABLESAW_SCREEN_HANDLER);
+		
+		Registry.register(Registries.BLOCK, new Identifier(MODID, "tablesaw"), TABLESAW);
+		Registry.register(Registries.ITEM, new Identifier(MODID, "tablesaw"), 
+				new BlockItem(TABLESAW, new QuiltItemSettings()));
+		ItemGroupEvents.modifyEntriesEvent(ItemGroups.FUNCTIONAL_BLOCKS).register(entries -> entries.addItem(TABLESAW));
+		
+		//Registry.register(Registries.SOUND_EVENT, TableSaw.TABLESAW_SFX, TABLESAW_SOUND_EVENT);
```

Comment:
Why is this commented out?

Reply:
I wasn't sure of the 1.19.4 equivalent for sound events.

Reply:
Resolved with https://github.com/DebuggyTeam/tablesaw/pull/22/commits/510ff42f5d04bc2aa75bb94980d1071633385a5e